HerPublicWebsite.BusinessLogic/Services/CsvFileCreator/CsvFileCreator.cs

```diff
@@ -272,6 +272,8 @@ public CsvRowReferralRequest(ReferralRequest request)
                 IncomeBand.GreaterThan31000 => "£31k or above",
                 IncomeBand.UnderOrEqualTo34500 => "Below £34.5k",
                 IncomeBand.GreaterThan34500 => "£34.5k or above",
```

Comment:
Same here re: we're keeping these around for backwards-compatibility with old referrals